### PR TITLE
Updated and rewrited language files

### PR DESCRIPTION
**Pull request by Myron472**
- Added translation for new strings;
- Line numbers in Russian and English language files now match;
- Rewritten some string in both English and Russian language files;
- Fixed typos;
- Added comments to context dependent or difficult cases.

Although Russian file is mostly translated it still has some strings I haven't translated for various reasons: some aren't because I didn't understand what it does, some aren't because I didn't find them in EinkBro and I don't understand in what context does it appear.

### CODE REVIEW

**Comment by Myron472** [comment]
Also why Build failed?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -56,77 +55,77 @@
...
 
     <string name="setting_title_location">Location</string>
     <string name="setting_summary_location">Allow websites to access your location. Some websites (like Google Maps for navigation)
         need this enabled.</string>
 
     <string name="setting_title_history">History</string>
-    <string name="setting_summary_history">Save the history of visited websites.</string>
-    <string name="save_history_mode_save_when_open">Save when opening</string>
+    <string name="setting_summary_history">Save the history of visited websites</string>
+    <string name="save_history_mode_save_when_open">Save when opening</string> <!-- When opening tab or EinkBro? -->
```
Comment (plateaukao) [inline app/src/main/res/values/strings.xml:112]: when opening a new tab, does EinkBro save that link to Tab List status right away (for next time launch), or save the  link until that tab is closed.

**Comment by plateaukao** [comment]
Hello, @Myron472 
Thank you for the Russian translations, and the refinement of English string version!
The build failure may be caused by key file decryption permission issue.
After merge, it should work (as long as all strings are still properly placed).

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Myron472) [reply app/src/main/res/values/strings.xml:112]: Thanks for clarification!
What do you think about changing _"Save when opening / closing"_ 2 strings to something more self-explanatory?
I would change them to either _"Save on tab opening"_ or _"Save when opening tab"_ and the same for _"Save when closing"_.

**Pull request merged**